.github/workflows/sync-docs.yaml

```diff
@@ -0,0 +1,43 @@
+name: Sync files to docs repo
+
+on:
+  push:
+    paths:
+      - tenzir.yaml.example
+      - web/openapi/openapi.yaml
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  sync-file:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout tenzir/tenzir repo
+        uses: actions/checkout@v4
+        with:
+          path: tenzir
+
+      - name: Checkout tenzir/docs repo
+        uses: actions/checkout@v4
+        with:
+          repository: tenzir/docs
+          token: ${{ secrets.TENZIR_DOCS_REPO_TOKEN }}
+          path: docs
+
+      - name: Copy file to docs repo
+        run: |
+          cp tenzir/tenzir.yaml.example docs/tenzir.yaml.example
+          cp tenzir/web/openapi/openapi.yaml docs/src/content/apis/openapi.node.yaml
+          cd docs
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git add tenzir.yaml.example
+          if ! git diff --cached --quiet; then
+            git commit -m "Sync files from tenzir/tenzir"
```

Comment:
For debugging purposes it'd be a good idea to include the previous and the current commit hash. There's `${{ github.event.push.after }}` and `${{ github.event.push.before }}` for this[^1].

[^1]: https://docs.github.com/en/webhooks/webhook-events-and-payloads#push

Reply:
Do you mean something like this?

```sh
    # Extract short commit SHAs
    SHORT_BEFORE=$(echo "${{ github.event.push.before }}" | cut -c1-7)
    SHORT_AFTER=$(echo "${{ github.event.push.after }}" | cut -c1-7)

    if ! git diff --cached --quiet; then
      git commit -m "Sync files from tenzir/tenzir (${SHORT_BEFORE}..${SHORT_AFTER})"
      git push
    else
      echo "No changes to commit."
    fi
```

Reply:
We use a length of 10 for commit hashes in all other places, so I'd recommend using that, but it captures the idea perfectly.